lib/repo/subversion_repository.rb

```diff
@@ -201,6 +201,48 @@ def stringify_files(files)
     end
     alias download_as_string stringify_files # create alias
 
+    # Generate and write the SVN authorization file for the repo.
+    def self.__generate_authz_file
+      return true if !MarkusConfigurator.markus_config_repository_admin?
+      valid_groupings_and_members = {}
+      assignments = Assignment.all
+      assignments.each do |assignment|
+        valid_groupings = assignment.valid_groupings
+        valid_groupings.each do |gr|
+          accepted_students = gr.accepted_students
+          accepted_students = accepted_students.map { |student| student.user_name }
+          valid_groupings_and_members[gr.group.repo_name] = accepted_students
+        end
+      end
+      tas = Ta.all
+      tas = tas.map { |ta| ta.user_name }
+      admins = Admin.all
+      admins = admins.map { |admin| admin.user_name }
+      tas_and_admins = tas + admins
+      invalid_groups = Group.all
+      invalid_groups = invalid_groups.map { |group| group.repository_name }
+      authz_string = ""
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.